### PR TITLE
add docs dir with Table of Contents

### PR DESCRIPTION
**Pull request by ORESoftware**
please add docs dir with Table of Contents

### CODE REVIEW

File: docs/table-of-contents.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+# Material2 Table of Contents
```
Comment (Chan4077) [inline docs/table-of-contents.md]: Shouldn't this be "Material" or just "Table of Contents"?

Comment (ORESoftware) [reply docs/table-of-contents.md]: yeah, something like that, either way, a docs folder or examples folder would help new users

Comment (donroyco) [reply docs/table-of-contents.md]: So your PR is not ready yet?

Comment (ORESoftware) [reply docs/table-of-contents.md]: this PR is just a concept, if you like the idea, I can fill it out

**Comment by jelbourn** [comment]
Our docs live on material.angular.io; we shouldn't need a TOC for markdown files on github.

**Pull request closed without merging**